api/api/src/test/scala/uk/ac/wellcome/platform/api/images/ImagesTest.scala

```diff
@@ -73,6 +73,36 @@ class ImagesTest extends ApiImagesTestBase with ElasticsearchFixtures {
     }
   }
 
+  it("returns matching results when using workdata from the redirected work") {
+    withApi {
+      case (ElasticConfig(_, imagesIndex), routes) =>
+        val baguetteImage = createAugmentedImageWith(
+          imageId = Identified("a", createSourceIdentifier),
+          parentWork = createIdentifiedWorkWith(title = Some("Baguette is a French bread style"))
+        )
+        val focacciaImage = createAugmentedImageWith(
+          imageId = Identified("b", createSourceIdentifier),
+          parentWork = createIdentifiedWorkWith(title = Some("A Ligurian style of bread, Focaccia"))
+        )
+        val schiacciataImage = createAugmentedImageWith(
+          imageId = Identified("c", createSourceIdentifier),
+          parentWork = createIdentifiedWorkWith(title = Some("Schiacciata is a Tuscan focaccia")),
```

Comment:
Actually I didn't mean to knock off mantou, it's just that because Ligurian focaccia was there I felt I had to add the Tuscan one. You know, regional rivalries and stuff... 😬 😁 
You're probably right about noodles

Reply:
ha, you must be the first Italian to say that...